endpoints/openrtb2/auction.go

```diff
@@ -354,12 +374,20 @@ func sendAuctionResponse(
 	account *config.Account,
 	labels metrics.Labels,
 	ao analytics.AuctionObject,
+	seatNonBid *openrtb_ext.NonBidCollection,
 ) (metrics.Labels, analytics.AuctionObject) {
 	hookExecutor.ExecuteAuctionResponseStage(response)
 
+	stageOutcomes := hookExecutor.GetOutcomes()
+	seatNonBid.Append(getNonBidsFromStageOutcomes(stageOutcomes))
+	ao.SeatNonBid = seatNonBid.Get()
+
 	if response != nil {
-		stageOutcomes := hookExecutor.GetOutcomes()
 		ao.HookExecutionOutcome = stageOutcomes
+		err := setSeatNonBidRaw(ao.RequestWrapper, response, ao.SeatNonBid)
```

Comment:
`RequestWrapper` is not being passed to `sendAuctionResponse` currently. We will have to change   `sendAuctionResponse` definition to accept `RequestWrapper` as argument instead of `BidRequest`.